opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -2454,15 +2517,6 @@ namespace detail {
 #endif
         }
 
-        // compute PAV and PORV for every regions.
```

Comment:
This calculation can be done locally (on the local grid), as long as the hydrocarbon porevolume for the domain/local grid is passed along with weighted pressure averages to the process doing the final output. I am happy to explain in detail on a hangout.